location_summary_plots.ipynb

```diff
@@ -270,9 +75,9 @@
     "        .agg({aggr_var: [\"min\", \"mean\", \"max\"]})\n",
     "    )\n",
     "    \n",
-    "    if len(np.unique(temp_df[\"mean\"])) < 5:\n",
-    "        print(f\"No meaningful variation, skipping location: {' '.join(title.split()[-2:])}, var: {index}\")\n",
-    "        return \"skip\"\n",
+    "    # if len(np.unique(temp_df[aggr_var])) < 5:\n",
+    "    #     print(f\"No meaningful variation, skipping location: {' '.join(title.split()[-2:])}, var: {index}\")\n",
+    "    #     return \"skip\"\n",
```

Comment:
Same as my other comment re: commented out code. Can this commented out code be removed? If not, I'd suggest adding a comment above it to indicate why/when it should be uncommented.

Reply:
Thanks for the catches!